### ISSUE TITLE
Keyboard Nav doesn't work properly when zoomed in

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/gravity-force-lab-basics#313]
The fix for the issue that was caught in https://github.com/phetsims/balloons-and-static-electricity/issues/546 should be cherry-picked into the 1.1 release of GFL:B as wel.
